infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -721,6 +712,7 @@ synchronized SegmentHandle getSegmentHandleForAddress(long address) {
                 }
 
                 FileChannel fc1 = getChannel(a, false);
```

Comment:
Can you add a comment about what these file channels are (or, rename them?) fc1a isn't really meaningful...